src/actions/workplace/README.md

```diff
@@ -0,0 +1,14 @@
+# Workplace by Facebook
+## Send a message to a Workplace by Facebook group or member referencing Looker data
+
+This action will send a message and link to a Workplace by Facebook group or member through a custom Looker integration created in Workplace by Facebook.
+
+1. Go to your Workplace by Facebook administration page [e.g. something of the format https://YOURCOMPANY.facebook.com/work/admin/?section=apps&tab=custom].
```

Comment:
these instructions need to be updated.

Reply:
There are updated instructions in the latest code...